.travis.yml

```diff
@@ -24,39 +24,6 @@ addons:
       - fonts-unfonts-core
 jobs:
   include:
-    - stage: tests
-      name: "Lint & Accessibility"
-      script:
-      - npm run build
-      - npm run lint
-      - npm run test:axe
-    - script:
-      - npm run build
-      - |
-        if [ $TRAVIS_PULL_REQUEST != false ] && [ $TRAVIS_SECURE_ENV_VARS == true ]; then
-            echo "Pull request with secure environment variables, running BrowserStack tests...";
-            npm run test:bs || travis_terminate 1;
-        else
-            echo "Not a pull request and/or no secure environment variables, running headless tests...";
-            npm run test:headless || travis_terminate 1;
-        fi
-      name: "BrowserStack"
-      env:
-        - BROWSERSTACK_USERNAME: margareepeacocke1
-        # BROWSERSTACK_ACCESS_KEY
-        - secure: LLdLul0G53je1zR+Mkn3KGJg3PXco6cgHcjElAJS1ZlA8bT2TGLmvdv3rAbqKG6Ff1LOtEViFJxXT/A31mu2wxEZl0xcZE4Vu17Ykz9n9SOl0UIFyBWruKj0g7H34eQyKL1yBZ5XUuiE5qLgJSepyrYF4v25X3cwojx4HEDw88bGzTJLuazPS9SZc5FspP/EFgvkXTGC/FWqEURDIJtZedvtLlGWsb/dZ6O2d8EGhZFYtnDNPSUhUSC2t/xkwxd/+zkj7ooci1cWBNrETBGKFhSKHdA46FyPz437siDa4nG5KL8an9wVND9sZ1LaF/CC1zDA902iY9SxP5rrRehrBqaQg+M3GOxs2yTJJqNM7fm7ws5MHKeaF2pukXkj+CR6vCOIV/2sGCOa0lwFGrV7EQ77oeVKS1q6iSz/5QJNGIppAoNKRSE604Pl7l7WEyUY4KJ/TVtWiPSTtpn4SMmpofIF3N0qtUIy2cXb+6MVhp7Tgo4MeErWKjplMdXC9ZkGtGw407wKX6oCtof74kCzkWQ3W+gAc38/Uvz7P1ky/vbxz4OPmBNL8cCBFnWRciWO8P5rcKgWHLGjkn7CMwYwjSu8QwQW+aWA/9c0sA98SoVz9TUEnG3l/ZJozkSpzH7n+T8iwY+T8F3tXPn6Ycdjrip/N6A/LKy7rSZi+zbQvKY=
-    - script:
-      - |
-        if [ $TRAVIS_PULL_REQUEST != false ] && [ $TRAVIS_SECURE_ENV_VARS == true ]; then
-            echo "Pull request with secure environment variables, running Sauce tests...";
-            npm run build;
-            npm run test:sauce || true;
-        fi
-      name: "SauceLabs"
-      env:
-        - SAUCE_USERNAME: Gaudi1
-        # SAUCE_ACCESS_KEY
-        - secure: bwPUP8HCJ4x/VQHYT83fw28MF6fZqcCFYYLQIuYl5d8QXPuHNiON+36YSEkIix/2AjuQR8OtNqy7MPem013QUzEl9ayJ+P/Kto3hN84D6fNj4DXKUtDywDeAs6B0hv3OZZIUJ2ixqClyGDRoK9M51NzS3d9Nts8rEDTJY5CqnD7s+0mOUXfmr7ehStQfs4JdsGApdp5ByA7btC6t3vNAUSoGOlnfKGb4VP/cCXAS64Suy/8Q09fCCKr2BMswQwR6hwb52EJ50W9JbpXu/S0DwrwwM5ZGCG4+SSXqrMoaWXeoOjjQ+tdTx+Rj/cvYO0IE9vJvEMubFCzjs57d2r9IwoKmJg27Fh7YPlZk4YEhcge2vDO9AmlGKGDKeEYco4fXelAmmCcFjmrWXjH8Uc31n2wZj4yYg6AKU2M+GvMYNmh+aI1qGJ+urpQigYdf8IWpqryW19wiOzRm+N6JhdFGlfgnJMEv/OmdTwGOXKA1Qlmc0Bv8ZjWHa6bVMrwnueygoUwc7rWYQg0SjIaW8zhTWwy2JkBoCD1/TpKYTNOdL4VSaxpRSCQn4uAlJ2rNBIyv7FM0S7ycKMTVQGXIyaRo28vKUEGUDYGYWbu++5fBxRfaHWeE79ZFE823LyFQetTUmr7ms/i4pqkaM/m+q840s34lLwmupSoUoVBAGezVsk8=
```

Comment:
For some reason this Travis config ends up running on NPM `7.0.8` which results in a bunch of visual diffs whereas [a fresh PR](https://github.com/BrightspaceUI/core/pull/939) runs on `6.14.8`. So confused...